src/governments.txt

```diff
@@ -0,0 +1,491 @@
+	claim_states = no
+	religion = no
+	republic = no
+	dictatorship = no
+	is_elective = no
+	queen = no
+	heir = no
+	has_parliament = no
+	has_devotion = no
+	has_meritocracy = no
+	allow_force_tributary = yes
+	duration = 0
+	election_on_death = no
+	monarchy = no
+	tribal = no
+	different_religion_acceptance = 0
+	different_religion_group_acceptance = 0
+	boost_income = no
+	monastic = no
+	can_use_trade_post = no
+	native_mechanic = no
+	can_form_trade_league = no
+	free_city = no
+	is_trading_city = no
+	trade_city_reform = ""
+	maintain_dynasty = no
+	allow_migration = no
+	nation_designer_trigger = { }
+	nation_designer_cost = 0
+	papacy = no
+	has_harem = no
+	has_pashas = no
+	has_janissaries = no	
+	allow_vassal_war = no
+	allow_vassal_alliance = no
+	min_autonomy = 0
+	factions_frame = 2
+	factions = { }
+	foreign_slave_rulers = no
+	royal_marriage = yes
+	nomad = no
+	raze_province = no
+	assimilation_cultures = { }
+	states_general_mechanic = { }
+	valid_for_nation_designer = yes
+	allow_normal_conversion = yes
+	force_conversion_gives_global_holy_war_released_modifier = no
+	start_territory_to_estates = 0.75
+	has_term_election = no
+	force_admiral_leader = no
+	force_general_leader = no
+	admirals_become_rulers = no
+	generals_become_rulers = no
+	allow_banners = no
+	uses_revolutionary_zeal = no
+	revolutionary = no
+	revolutionary_client_state = no
+	allow_draft_transport_ships = no
+	free_concentrate_development = no
+	disallowed_trade_goods = {}
+	can_customise_heir = yes
+	allow_cawa = no
+	can_change_primary_culture = yes	
+	allow_carolean = no	
+	can_inherit_personal_unions = yes
+	block_cultural_union = no
+	has_cultural_union = no 
+	can_remove_idea_group = yes
+	mercs_do_not_cost_army_professionalism = no	
+	
+	government_abilities = { 
+		magicka_mechanic
+	}
+}
+
+new government_names default_monarchy = {
+	rank = {
+		1 = "Barony"
+		2 = "County"
+		3 = "Duchy"
+		4 = "Kingdom"
+		5 = "High Kingdom"
+		6 = "Empire"
+	}
+	ruler_male = {
+		1 = "Baron"
+		2 = "Count"
+		3 = "Duke"
+		4 = "King"
+		5 = "High King"
+		6 = "Emperor"
+	}
+	ruler_female = {
+		1 = "Baroness"
+		2 = "Countess"
+		3 = "Duchess"
+		4 = "Queen"
+		5 = "High Queen"
+		6 = "Empress"
+	}
+	consort_male = {
+		1 = "Baron"
+		2 = "Count"
+		3 = "Duke"
+		4 = "King"
+		5 = "High King"
+		6 = "Emperor"
+	}
+	consort_female = {
+		1 = "Baroness"
+		2 = "Countess"
+		3 = "Duchess"
+		4 = "Queen"
+		5 = "High Queen"
+		6 = "Empress"
+	}
+	heir_male = {
+		1 = "Prince"
+		2 = "Prince"
+		3 = "Prince"
+		4 = "Prince"
+		5 = "Prince"
+		6 = "Prince"
+	}
+	heir_female = {
+		1 = "Princess"
+		2 = "Princess"
+		3 = "Princess"
+		4 = "Princess"
+		5 = "Princess"
+		6 = "Princess"
+	}
+	trigger = {
+		government = monarchy
+	}
+}
+new government_names default_republic = {
+	rank = {
+		1 = "Republic"
+		2 = "Republic"
+		3 = "Republic"
+		4 = "Grand Republic"
+		5 = "Grand Republic"
+		6 = "Empire"
+	}
+	ruler_male = {
+		1 = "Lord"
+		2 = "Lord"
+		3 = "Lord"
```

Comment:
Suggesting Syndic in the lowest to second lowest as the ruler of a city and outside of it